catalog/spatial/registry/registry-api-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -34,7 +34,34 @@
 
     <reference id="configurationAdmin" interface="org.osgi.service.cm.ConfigurationAdmin"/>
 
-    <bean id="metacardMarshaller" class="org.codice.ddf.registry.schemabindings.helper.MetacardMarshaller">
+    <reference id="federationAdminService"
+               interface="org.codice.ddf.registry.federationadmin.service.FederationAdminService"/>
+    <reference id="registryPublicationService"
+               interface="org.codice.ddf.registry.federationadmin.service.RegistryPublicationService"/>
+
+    <bean id="registryAutoPublisher"
+          class="org.codice.ddf.registry.api.impl.RegistryStoreAutoPublisher">
+        <property name="federationAdminService" ref="federationAdminService"/>
+        <property name="registryPublicationService" ref="registryPublicationService"/>
+    </bean>
+
+    <reference-list id="registryAutoPubList" interface="org.codice.ddf.registry.api.RegistryStore"
+                    availability="optional">
+        <reference-listener ref="registryAutoPublisher" bind-method="bindRegistryStore"/>
+    </reference-list>
+
+    <service ref="registryAutoPublisher" interface="org.osgi.service.event.EventHandler">
+        <service-properties>
+            <entry key="event.topics">
+                <array value-type="java.lang.String">
+                    <value>org/osgi/framework/ServiceEvent/MODIFIED</value>
```

Comment:
I'm not going to say this is the wrong event to listen to/way to do this, but this would be the first event handler in DDF listening to a native OSGi service event.